AirCasting/Utils/Strings.swift

```diff
@@ -305,300 +439,501 @@ struct Strings {
+    }
+    
     enum SDSyncCompleteView {
-        static let title: String = LocalizedStringKey("Sync complete").stringKey
-        static let message: String = LocalizedStringKey("The data from your AirBeam3 was synchronized successfully to the app. The SD card was cleared.").stringKey
-        static let SDClearTitle: String = LocalizedStringKey("SD card cleared").stringKey
-        static let SDClearMessage: String = LocalizedStringKey("SD inside your AirBeam was cleared sucesfully").stringKey
-    }
-
+        static let title: String = NSLocalizedString("Sync complete",
+                                                     comment: "")
+        static let message: String = NSLocalizedString("The data from your AirBeam3 was synchronized successfully to the app. The SD card was cleared.",
+                                                       comment: "")
+        static let SDClearTitle: String = NSLocalizedString("SD card cleared",
+                                                            comment: "")
+        static let SDClearMessage: String = NSLocalizedString("SD inside your AirBeam was cleared sucesfully",
+                                                              comment: "")
+    }
+    
     enum ClearingSDCardView {
-        static let title: String = LocalizedStringKey("Clearing SD card").stringKey
-        static let message: String = LocalizedStringKey("This should take less then 10 seconds.").stringKey
-        static let failedClearingAlertTitle: String = LocalizedStringKey("Failed to clead SD card").stringKey
-        static let failedClearingAlertMessage: String = LocalizedStringKey("Try again later").stringKey
-    }
-
+        static let title: String = NSLocalizedString("Clearing SD card",
+                                                     comment: "")
+        static let message: String = NSLocalizedString("This should take less then 10 seconds.",
+                                                       comment: "")
+        static let failedClearingAlertTitle: String = NSLocalizedString("Failed to clead SD card",
+                                                                        comment: "")
+        static let failedClearingAlertMessage: String = NSLocalizedString("Try again later",
+                                                                          comment: "")
+    }
+    
     enum DefaultDeleteSessionViewModel {
-        static let all: String = LocalizedStringKey("All").stringKey
+        static let all: String = NSLocalizedString("All",
+                                                   comment: "")
     }
-
+    
     enum DeviceHandler {
-        static let alertTitle = LocalizedStringKey("Not supported device").stringKey
-        static let alertMessage = LocalizedStringKey("To use the AirBeam3 in standalone mode and sync the SD card, an iPhone8 or higher is required").stringKey
+        static let alertTitle = NSLocalizedString("Not supported device",
```

Comment:
a bit funky wording here. Maybe we should change to something better like `unsupprted device` or `this device is not supported`

Reply:
Sure, changing!